config/locales/fr.yml

```
@@ -760,6 +770,7 @@ fr:
       search_prompt: Rechercher dans %{portal_name} ...
       submit_feedback: Soumettre un retour d'expérience
       support: Support
+      agents : Agents
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)